### ISSUE TITLE
Random mob support

### ISSUE THREAD

**Issue by timbru31** [timbru31/SilkSpawners#129]
Originally reported from _Shano_dekono_ via the SpigotMC forums
>I have a suggestion to add to the plugin, when crafting spawn cages if config is set to "defaultCreature: random" it will spawn random entity mob cages (creepers,skeletons,zombies etc) instead of set mobs.

**Comment by axemwa** [comment]
Yeah random mobs option would be handy thanks for considering it :D
